js/Components/NavBar/newsViewer.js

```diff
@@ -0,0 +1,29 @@
+import { startProgress, stopProgress } from "../NewsList/progress.js";
+
+const $listButton = document.querySelector(".news-navbar_content__list");
+const $gridButton = document.querySelector(".news-navbar_content__grid");
+const $listSection = document.querySelector(".news-section-list");
+const $gridSection = document.querySelector(".news-section-grid");
+
+const clickList = () => {
+  $listButton.childNodes[1].childNodes[1].setAttribute("fill", "#4362D0");
+  $gridButton.childNodes[1].childNodes[1].setAttribute("fill", "#879298");
```

Comment:
이부분 동일하게 사용되니까 전역으로 뺴도 좋을것 같아요~

```suggestion
  const $listElem =   $listButton.childNodes[1].childNodes[1]
  const $gridElem =  $gridButton.childNodes[1].childNodes[1]
```